app/src/main/res/layout/user_id.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
`user_id.xml`, `user_id_error.xml` and `version.xml` appear to be more or less identical - should we re-use the layout?